main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/t-eckert/shepherd/cmd"
+	"github.com/t-eckert/shepherd/github"
+)
+
+func main() {
+	args := os.Args
+
+	command, err := cmd.NewCommand(args)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	issues, err := github.FetchIssues(command.Origin)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	for _, issue := range issues {
+		log.Printf("Issue #%d\t%s", issue.Number, issue.Title)
+	}
+
+	// Modify the prepend (e.g. helm:) if requested
+	if prepend, ok := command.Flags["modify-prepend"]; ok {
+		// Remove `:` at the end of prepend if user passed it in by accident
+		if prepend[len(prepend) - 1:] == ":" {
+			prepend = prepend[0:len(prepend) - 1]
+		}
+
+		// Get user confirmation before proceeding
+		shouldPrepend := askToContinue(fmt.Sprintf("Modify %d issues with prepend `%s:`?", len(issues), prepend))
+		if !shouldPrepend {
+			safeAbort("Stopping without modifying issue titles.")
+		}
+
+		if err := github.ModifyPrepend(issues, command.Origin, prepend); err != nil {
+			log.Fatalln(err)
+		}
+
+		// Refetch the updated issues
+		issues, err = github.FetchIssues(command.Origin)
+		if err != nil {
+			log.Fatalln(err)
+		}
+		for _, issue := range issues {
+			log.Printf("Issue #%d\t%s", issue.Number, issue.Title)
+		}
+
+		log.Printf("Modified %d issues", len(issues))
```

Comment:
Technically there's a race condition here: if an issue is created in-between your first fetch and this fetch then you won't have actually modified that issue. Not likely to trigger this though since the script runs so quickly.